tests/__init__.py

```diff
@@ -12,7 +12,10 @@
 except ImportError:
     resource = None
 import signal
-import subprocess
+try:
+    import subprocess32 as subprocess  # py2
```

Comment:
While `run_python` would definitely benefit from timeout, as of now it breaks Python2.6 tests. I suggest to retract subprocess32 for now, if a simple solution is not possible, to avoid blocking of merging this threading patch.

If you really insist on subprocess32, it would safely work backwards:
```
import subprocess
if six.PY2:
  # temporary workaround `run_python` needs `.communicate()` with timeout
  import subprocess32
elif six.PY3:
  subprocess32 = subprocess
```